js/components/moreOptionsPanel.js

```diff
@@ -68,6 +74,11 @@ var MoreOptionsPanel = React.createClass({
           onMouseOver={this.highlight} onMouseOut={this.removeHighlight}/>
       </a>,
 
+      "audioAndCC": <a className="oo-multiaudio oo-control-bar-item" onClick={this.handleMultiAudioClick} key="audioAndCC">
+        <Icon {...this.props} icon="audioAndCC" style={buttonStyle}
+              onMouseOver={this.highlight} onMouseOut={this.removeHighlight}/>
+      </a>,
+
       "closedCaption": <a className="oo-closed-caption oo-control-bar-item" onClick={this.handleClosedCaptionClick} key="closedCaption">
```

Comment:
Thank you for your changes! Do we need to add tests for that case?

Reply:
\+